commit 05f03f0db2ed93d1f9296ee721ec47c6f53a2be7 (refs/changes/76/5676/2)
Author: Rui Loura <rui@joyent.com>
Date:   2019-02-28T23:24:42+00:00 (7 months ago)
    
    TRITON-1271 vmapi vms.full test should account for a negative number in available RAM
    Reviewed by: Cody Peter Mello <melloc@writev.io>
    Approved by: Cody Peter Mello <melloc@writev.io>

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index c3112de..f1712dd 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2350,7 +2350,7 @@ exports.resize_package_up_fail = function (t) {
                         'error code should be InsufficientCapacity');
                     t.ok(error.message.match(
                         'Required additional RAM \\(\\d+\\) ' +
-                        'exceeds the server\'s available RAM \\(\\d+\\)'),
+                        'exceeds the server\'s available RAM \\(-?\\d+\\)'),
                         'error message should explain additional RAM required');
 
                     cb();
